### ISSUE TITLE
Syncgroup Priority is 0 for Syncgroup creator

### ISSUE THREAD

**Issue by afandria** [vanadium/issues#798]
Using Mojo + Syncbase and set the priority to 3.

**Comment by afandria** [comment]
@asadovsky @hpucha

**Comment by hpucha** [comment]
Is the plan to use the priority bit in the joiner info? Right now, we are not using that information anywhere in the sync logic.

**Comment by afandria** [comment]
Yes, that's right. While I don't expect `syncPriority: 3` to do anything yet, I used it to create the SyncgroupMemberInfo for both syncgroup joiners and the syncgroup creator. Because of that, I think the creator should get syncPriority 3, but the `getMemberInfo` shows it as having 0 instead.
